indigo_api/views/documents.py

```diff
@@ -354,7 +356,11 @@ def post(self, request, document_id):
             doc.frbr_uri = frbr_uri
             xml = doc.to_xml(encoding='unicode')
 
-        return Response({'output': xml})
```

Comment:
ideally the client should be extracting the eId and doing with it whatever it needs to, so these changes can fall away